internal/model/rwfolder.go

```diff
@@ -54,6 +54,19 @@ var (
 	errNoDevice = errors.New("no available source device")
 )
 
+const (
+	dbUpdateHandleDir = iota
+	dbUpdateDeleteDir
+	dbUpdateHandleFile
+	dbUpdateDeleteFile
+	dbUpdateShortcutFile
+)
+
+type dbUpdateJob struct {
+	file    protocol.FileInfo
+	jobType int
```

Comment:
Yeah, but that is also just as equivalent when taking a shortcut without actually changing the file content.